x/marker/types/events.go

```diff
@@ -33,6 +33,8 @@ const (
 	EventTelemetryKeyMint string = "mint"
 	// EventTelemetryKeyTransfer transfer telemetry metrics key
 	EventTelemetryKeyTransfer string = "transfer"
+	// EventTelemetryKeyIbcTransfer ibctransfer telemetry metrics key
```

Comment:
do we need this telemetry key? just asking

Reply:
I meant to add telemetry to match the regular send.  Added.